tests/test_endl.py

```diff
@@ -0,0 +1,58 @@
+import os
+import io
+import warnings
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+from math import e
+
+import numpy as np
+from numpy.testing import assert_array_equal, assert_allclose, \
+    assert_array_almost_equal
+
+from pyne.utils import QAWarning
+warnings.simplefilter("ignore", QAWarning)
+
+from pyne.endl import Library
+
+import nose
+from nose.tools import assert_equal
+
+def ignore_future_warnings(func):
+    """This is a decorator which can be used to ignore FutureWarnings
+    occurring in a function."""
+    def new_func(*args, **kwargs):
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore", category=FutureWarning)
+            return func(*args, **kwargs)
+    new_func.__name__ = func.__name__
+    new_func.__doc__ = func.__doc__
+    new_func.__dict__.update(func.__dict__)
+    return new_func
+
+def test_loadfile():
+    try:
+        assert(os.path.isfile('epdl97_eedl_Pb'))
+    except AssertionError:
+        try:
+            import urllib.request as urllib
+        except ImportError:
+            import urllib
```

Comment:
This import should happen at the top of the module and should use the `sys.version_info` check.  See other test modules for examples.